modules/tbprofiler.nf

```diff
@@ -41,9 +45,13 @@ process tbprofiler {
     tuple val(sample_id), path("${sample_id}_tbprofiler*.{json,csv}"), emit: reports
     tuple val(sample_id), path("${sample_id}_tbprofiler*.{bam,bam.bai}"), emit: alignment
     tuple val(sample_id), path("${sample_id}_tbprofiler*.vcf"), emit: variants
+    tuple val(sample_id), path("${sample_id}_tbprofiler_provenance.yml"), emit: provenance
     
     script:
     """
+    printf -- "- process_name: tb-profiler\\n" > ${sample_id}_tbprofiler_provenance.yml
```

Comment:
Would it be possible to collect the tbprofiler database version and include that in the pipeline provenance file?